README.md

```diff
@@ -61,4 +61,22 @@ If tests return session errors in relation to redis, like the following:
       + expected - actual
 ```
 
-Ensure that the correct redis password is set - `export FH_REDIS_PASSWORD=feedhenry101`
\ No newline at end of file
+Ensure that the correct redis password is set - `export FH_REDIS_PASSWORD=feedhenry101`
+
+# Known issues
+
+## Two sync loops per sync frequency
+Two sync loops may be made per sync frequency if the server-side sync frequency
+is different from the client-side frequency.
+
+This is because the client and server sync frequencies are set independently.
+Setting a long frequency on a client does not change the sync frequency on the
+server.
+
+To workaround this set each datasets `syncFrequency` value on the server
+to (or near to) the `sync_frequency` value of the corresponding on the 
```

Comment:
can we clarify what 'or near to' means?
Should it always be less than/greater than something?